common/appointment/tracking.ts

```diff
@@ -29,15 +30,21 @@ export async function trackUserJoinAppointmentMeeting(user: User, appointment: A
 
         logger.info(`Tracked User(${user.userID}) joining Appointment(${appointment.id}) for Subcourse(${subcourse.id})`);
     } else if (appointment.matchId) {
+        const match = await getMatch(appointment.matchId);
```

Comment:
??

Reply:
we miss the name of the match partner here.
![missing_name](https://github.com/corona-school/backend/assets/115979086/2ee343b6-31e2-4423-88fe-dabf7878d2ef)

Reply:
Thanks @LomyW for fixing this

Reply:
I think I'm too tired to do reviews in the evening. I could swear that I did not see any usage of the match variable 😅

Reply:
Thank you for fixing it 👍